index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="container">
+        <a href="#" class="name-info">Sunday Stanley</a>
+        <div class="menu-toggle">
+          <img src="./image/menu.png" alt="menu bar" />
+        </div>
+        <ul class="list-container">
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href=""> About </a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+          <li>
+            <a href=""><img src="./image/msg.png" alt="msg icon" /></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="bio-container">
+        <h1>Hey There. I’m Stanley</h1>
+        <h3>I’am a Software Developer</h3>
+        <div class="paragragh">
```

Comment:
[REQUIRED] Please Avoid skipping heading levels: always start from h1, followed by h2, and so on.
so your code became more semantically accurate.

Reply:
checked